test/test_helper.rb

```diff
@@ -300,6 +300,7 @@ def mappings_wrapper(type, body)
       }
     }
     mapping[type] = body
+    if disable_all then mapping[type]["_all"] = { "enabled": false } end
```

Comment:
`_all: { enabled: false }` is not valid from ES 7 - https://www.elastic.co/guide/en/elasticsearch/reference/7.17/breaking-changes-7.0.html#all-meta-field-removed